src/jrd/Mapping.cpp

```diff
@@ -1171,7 +1171,9 @@ class SysPrivCache : public PermanentStorage
 					g |= l;
 				}
 
-				ULONG gg = 0; g.store(&gg);
+				FB_UINT64 gg = 0;
+				fb_assert(sizeof(gg) >= g.BYTES_COUNT);
```

Comment:
`static_assert` would be better here because none side of the comparison can be changed in runtime.

Reply:
done